File: src/test/programmers/level1/NumberOfPAndYInStrings.java
package test.programmers.level1;

import java.util.Locale;

public class NumberOfPAndYInStrings {
    public static void main(String[] args) {
        System.out.println(solution("Pyy"));
    }

    static boolean solution(String s) {
        boolean answer = true;
        int count = 0;

        String str = s.toLowerCase(Locale.ROOT);
        

        for(int i = 0; i < str.length(); i++) {
            char ch = str.charAt(i);

            if(ch == 'p') count++;
            if(ch == 'y') count--;
        }

        return answer = count == 0 ? true : false;
    }
}

File: src/test/programmers/python/level1/NumberOfPAndYInStrings.py
def solution(s):
    # count = 0
    # for i in range(len(s)):
    #     if s[i] == 'p' or s[i] == 'P':
    #         count += 1
    #     elif s[i] == 'y' or s[i] == 'Y':
    #         count -= 1
    # return True if count == 0 else False
    return s.lower().count('p') == s.lower().count('y')
    # count 함수를 활용해서 두 문자의 갯수가 같으면 True 아니면 False 를 반환한다.




